CONTRIBUTING.md

```diff
@@ -0,0 +1,132 @@
+# Contributing to the documentation
+
+## Repository structure
+
+Briefly, this is what you need to know about the documentation repository structure before contributing:
+
+* The documentation is module-based, which means most sections are individual modules. These modules are available in the */docs/topics/modules/* folder.
+* The modules are linked together to form a chapter, document, or guide using assemblies. These assemblies are asciidoc files in the */docs/topics/* folder. For example, the */docs/topics/getting-started-guide.adoc* file is the main assembly for the Getting Started Guide and links to other, smaller assemblies for parts or chapters within the guide.
+* Module files naming conventions closely follow the title of the corresponding section. For example, a section in the Getting Started document with the title *Working with pipelines* is named **working_with_pipelines.adoc** in the *modules* folder. In some cases, this is not a direct match, so to confirm the name of the asciidoc file for a section, click the section title and the ID after the *#* in the URL is the name of the module file. For example, a URL such as https://docs.openshift.io/getting-started-guide.html#viewing_build_pipeline_oso has a slightly different title but the ID after the hash symbol (**viewing_build_pipeline_oso**) is the name of the file.
+
+## Conventions and style guidelines
+
+To get started quickly, we recommend reviewing the following conventions that are relevant for this documentation suite:
+
+**Writing conventions**
+
+* For instructions, state the location, then action and subject. For example:
+  * Incorrect: *Click OK in the XYZ tab to continue.*
+  * Correct: *In the XYZ tab, click OK to continue.*
+  * Use gerunds in titles when possible. For example:
+    * Incorrect: *1.1 Create a new project*
```

Comment:
Typo: use gerunds is a first level bullet. GH does not let me edit while adding comments.
Ideally would be great to use a table here, but for now this is just fine.